### PR TITLE
Artist invite serve pages

### PR DESCRIPTION
**Pull request by Justin-Holmes**

### CODE REVIEW

File: apps/ello_serve/web/router.ex [hunk 1]
```diff
@@ -35,6 +35,7 @@ defmodule Ello.Serve.Router do
     get "/discover/all",          CategoryController, :index
     get "/discover/:category",    DiscoverPostController, :category
     get "/search",                SearchController, :index
+    get "/artist_invites",        ArtistInviteController, :index
```
Comment (alanpeabody) [inline apps/ello_serve/web/router.ex]: We should check with @rynbyjn, but I think maybe this is a dash not an underscore.

Comment (rynbyjn) [inline apps/ello_serve/web/router.ex]: @alanpeabody you are right. I think it was categories that started that trend.

File: apps/ello_serve/web/controllers/webapp/artist_invite_controller.ex [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+  def index(conn, _) do
+    render_html(conn, %{
+      artist_invites: fn -> artist_invites(conn) end
+    })
+  end
+
+  def show(conn, %{"id" => id_or_slug}) do
+    render_html(conn, %{
+      artist_invite: fn -> artist_invite(conn, id_or_slug) end,
```
Comment (alanpeabody) [inline apps/ello_serve/web/controllers/webapp/artist_invite_controller.ex]: I think we should probably treat the show page like we do the user show and post show - always find the invite and return 404 if it doesn't exist. That ensure we get legit 404s for google.

File: apps/ello_serve/web/templates/webapp/artist_invite/noscript.html.eex [hunk 3]
```diff
@@ -0,0 +1,55 @@
+<%= if assigns[:artist_invites] do %>
```
Comment (alanpeabody) [inline apps/ello_serve/web/templates/webapp/artist_invite/noscript.html.eex]: why do we need this conditional?

Comment (alanpeabody) [inline apps/ello_serve/web/templates/webapp/artist_invite/noscript.html.eex]: Oh, both show and index in the same view/template - got it.

Comment (alanpeabody) [inline apps/ello_serve/web/templates/webapp/artist_invite/noscript.html.eex]: You could consider using different controllers for this if you wanted. I know how this ended up set up wasn't perfect with one template per controller.

File: apps/ello_serve/web/templates/webapp/artist_invite/noscript.html.eex [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  <p><%= @artist_invite.raw_description %></p>
+  <%= for guide_section <- @artist_invite.guide do %>
+    <h3><%= guide_section["title"] %></h3>
+    <p><%= guide_section["raw_body"] %></p>
+  <% end %>
+
+  <%= if assigns[:selections] do %>
+    <h2>Selections</h2>
+    <%= for selection <- @selections do %>
+      <%= render PostView, "header.html", Map.put(assigns, :post, selection.post) %>
```
Comment (alanpeabody) [inline apps/ello_serve/web/templates/webapp/artist_invite/noscript.html.eex]: Can we render summary.html here instead of header/blocks/footer?

**Pull request merged**